<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<!-- Starting up nodes that give the pace to the inner controllers -->
	<node name="node_100Hz" pkg="crazyCmd" type="node_100Hz.py" output="screen"/>
	<node name="node_500Hz" pkg="crazyCmd" type="node_500Hz.py" output="screen"/>
	<node name="node_1000Hz" pkg="crazyCmd" type="node_1000Hz.py" output="screen"/>

    <group ns = "cf1">
		<node pkg="crazyCmd" type="dd_crazyflie_real_node.py" name="crazyflie_real_node" output="screen">
			<!-- <rosparam param="name">cf2</rosparam>
			<rosparam param="initial_position">[0.0, 0.0, 0.1]</rosparam> -->
		</node>
	</group>

    <!-- Starting up the node that perform all the operations for the path test task -->
    <!-- <node name="path_test_node" pkg="crazyCmd" type="path_test_node.py" output="screen"/> -->
</launch>